### DISCUSSION TITLE
Livelihood of outside the park limits

### DISCUSSION THREAD

**Discussion by karst** [OpenRCT2/OpenRCT2#17990] [Feature requests]
Currently the only thing that adds some livelihood to the outer rims of scenarios is the people walking to the park or if cheats are used someone could make a ride in test mode outside the park (but these can break down which is an issue). There are a few ways to make it more lively. 

@ZeeMajora made the suggestion of ambient rides, rides that are allowed to be placed during the scenario creation. I don't know what their intention on this was entirely but I could see them work without station and with teleporting points?

I was thinking of a bus that could come in and deliver peeps at a bus stop from which they would walk to the entrance. As well as an entrance point for cars to drive into a parking lot (perhaps to a path deadend/outlet), slowly fill up that lot, having peeps walking from the lot to the park. This would probably require a bit more advanced pathing AI (shortest way to an outlet and from there shortest way to the park entrance). 

![image](https://user-images.githubusercontent.com/1516893/189043904-d3966eb6-7cb1-477b-bd9d-c41761bd1151.png)

I think these things could overall really improve the livelyhood of the outer rims of parks which usually look abandoned or bare.

**Comment by J-PIE-314** [comment]
This is akin to the intended purpose of the car and bus shown in _[The Evolution of Roller Coaster Tycoon](https://web.archive.org/web/20210710210740/http://www.nicscorner.com/evo_of_rct.htm)_ article.
